lib/commands/generate.ts

```diff
@@ -7,15 +7,15 @@ import { default as config } from "./config";
 const command = {
 	aliases: ["g"],
 	command: "generate",
-	desc: "Generates new project",
+	desc: "generates custom template",
```

Comment:
I guess this can stay for now, though `generate` can later on be extended beyond just custom templates